components/automate-chef-io/content/docs/on-prem-builder.md

```diff
@@ -0,0 +1,93 @@
++++
+title = "Deploying Builder with Chef Automate"
+description = "use Chef Automate to deploy Builder"
+date = 2019-11-19T14:10:15-08:00
+weight = 20
+draft = false
+bref = ""
+toc = true
++++
+
+## Overview
+Enterprise customers may wish to set up an on-premises Habitat Builder depot to store
+Habitat packages for use by their own Habitat Studios and Supervisors. Use Chef Automate
+to deploy and manage an on-premises Habitat Builder depot.
+
+{{< warning >}}
+The current implementation of Automate-deployed Habitat Builder is an early milestone
+intended for use in proof-of-concept deployments. We do not consider it a fully-mature
+feature, and you should not either.
+{{< /warning >}}
+
+### System requirements
+Currently, Habitat Builder requires Chef Automate to be installed on the same host for
+authentication purposes. Outside the boundaries of a proof-of-concept exercise, we
+recommend against running installations of Chef Automate and Habitat Builder on
+the same host.
+
+#### Hardware
+The minimum with which Chef Automate and Habitat Builder can be deployed on a single host
+is:
+
+* 16 GB of RAM.
+* 130 GB of disk space, available to /hab.
+* 4 vCPUs.
+* Inbound network connectivity from LAN (HTTP/HTTPS) is required for internal clients to access the on-premises Chef Habitat Builder.
+
+For deployments that are expected to see production-scale workload, we recommend:
+* 64 GB of RAM.
+* 200 GB of diskspace, available to /hab.
+* 16 vCPUs.
+
+Roughly 80 GB of the disk space is designated for Chef Automate; the rest is used for
+Habitat Builder and the artifacts it stores. The current implementation uses Minio for
+Habitat artifact storage; we do not yet support using Artifactory for artifact storage.
+
+#### Operating system
+Chef Automate and Habitat Builder require
+
+* a Linux kernel of version 3.2 or greater
+* systemd as the init system
+* useradd
+* curl or wget
+* The shell that starts Automate should have a max open files setting of at least 65535
+
+### Deploy on-premises Habitat Builder
```

Comment:
We expect that most potential customers will be running this behind an airgap, so we should point to the directions for the airgap bundle.